### PR TITLE
Devcontainer GPU support #945

### PR DESCRIPTION
**Pull request by jacoverster**
# What does this PR do?
A small update to devcontainer.json to enable GPU support. Added "RunArgs" to devcontainer.json.

Fixes [issue 945](https://github.com/keras-team/keras-cv/issues/945)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/keras-team/keras-cv/blob/master/.github/CONTRIBUTING.md),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue? Please add a link
      to it if that's the case.
- [ ] Did you write any new necessary tests?
- [ ] If this adds a new model, can you run a few training steps on TPU in Colab to ensure that no XLA incompatible OP are used?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by bhack** [comment]
You need to sign the CLA

**Comment by jacoverster** [comment]
> You need to sign the CLA

Hi @bhack I signed.

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -32,6 +32,12 @@
 			"version": "os-provided"
 		}
 	},
+
+	// Uncomment this if GPU support it is required
```
Comment (bhack) [inline .devcontainer/devcontainer.json:36]: NIT: 
probably we could remove "it" also in the previous string.

**Review by bhack — APPROVED** [review]
It is ok for me.. we cannot test this in the CI cause it would require to run in cloudbuild as Gtihub Actions are CPU only.

But as it is commented by default as the other "optional" entry related to the GPU tensorflow docker image it is ok for me.

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (jacoverster) [reply .devcontainer/devcontainer.json:36]: Sure thing, I'll also update the top comment to reflect the same:

"// Uncomment this if GPU support is required"

**Comment by bhack** [comment]
Thanks

**Comment by bhack** [comment]
It don't know if you want to add a reference to https://github.com/microsoft/vscode-remote-release/issues/3972 in the source comments.

So that we could have a reminder to imporve this when we will have a little bit of scripting support instead of requiring to the developer to comment and uncomment multiple sections just to switch from cpu to gpu and back.

**Comment by jacoverster** [comment]
Good idea! I'll add a reference tomorrow morning 👍🏻 (it's evening here now)

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -32,6 +32,13 @@
 			"version": "os-provided"
 		}
 	},
+
+	// TODO: Improve to allow dynamic runArgs, see microsoft/vscode-remote-release#3972
+	// Uncomment this if GPU support is required
+	// "runArgs": [
+    //     "--gpus=all"
```
Comment (bhack) [inline .devcontainer/devcontainer.json:39]: Nit: can you align this comment

Comment (jacoverster) [reply .devcontainer/devcontainer.json:39]: That's so strange, in my working tree it looks correct:
image

Let me try editing it on GitHub.

Comment (jacoverster) [reply .devcontainer/devcontainer.json]: Ok, that seemed to do the trick. :)

**Review by ianstenbit — APPROVED** [review]
Thanks for the PR!

**Comment by jacoverster** [comment]
No problem, thanks for the accept and the great work you guys are doing.

**Pull request merged**